tests/simulation/test_simulation.py

```diff
@@ -259,17 +267,18 @@ def bell(a, b):
         #     _, _, cy = context_to_verilog_and_dump(get_context(triple_circle))
         #     f.write(str(cy))
         module, testbench = namespace_to_verilog(ns)
-        mod_path = Path(__file__).parent / "bell.sv"
-        tb_path = Path(__file__).parent / "bell_tb.sv"
-        with open(mod_path, mode="w") as f:
-            f.write(str(module))
-        with open(tb_path, mode="w") as f:
-            f.write(str(testbench))
-        cmd = iverilog.make_cmd(
-            "bell_tb",
-            [mod_path, tb_path],
-        )
-        # warnings.warn(cmd)
+        if self.args.write:
+            mod_path = Path(__file__).parent / "bell.sv"
```

Comment:
How does this syntax work?

Reply:
The `Path(__file__).parent / "bell.sv"`

Reply:
Take the parent of the current file then join with bell.sv